source/languages/en/riakcs/cookbooks/Using-Riak-CS-With-Keystone.md

```diff
@@ -47,35 +47,78 @@ based on user feedback, support may be added in the future.
 
 #### API
 
-Set the API using the `rewrite_module` configuration option. To use the
-S3 API, insert the following:
+Set the API using the `rewrite_module` configuration in the Riak CS
+`riak-cs.conf` file, or the old-style `app.config` file in the `riak_cs`
+section.
 
-```erlang
-{rewrite_module, riak_cs_s3_rewrite}
+To use the S3 API, insert the following:
+
+```riakcsconf
+rewrite_module = riak_cs_s3_rewrite
+```
+
+```appconfig
+{riak_cs, [
+           %% Other configs
+           {rewrite_module, riak_cs_s3_rewrite},
+           %% Other configs
+          ]}
 ```
 
 To use the OpenStack object storage API:
 
-```erlang
-{rewrite_module, riak_cs_oos_rewrite}
+```riakcsconf
+rewrite_module = riak_cs_oos_rewrite
+```
+
+```appconfig
+{riak_cs, [
+           %% Other configs
+           {rewrite_module, riak_cs_oos_rewrite},
+           %% Other configs
+          ]}
 ```
 
 #### Authentication Module
 
-Set the authentication module to specify the Keystone authentication
-module:
+Set the authentication module using the `auth_module` configuration in the Riak
+CS `riak-cs.conf` file, or the old-style `app.config` file in the `riak_cs`
+section.
+
+To specify the Keystone authentication module:
+
+```riakcsconf
+auth_module = riak_cs_keystone_auth
+```
 
-```erlang
-{auth_module, riak_cs_keystone_auth}
+```appconfig
+{riak_cs, [
+           %% Other configs
+           {auth_module, riak_cs_keystone_auth},
+           %% Other configs
+          ]}
 ```
 
 #### Operator Roles
 
-You may optionally override the default list of valid operator roles. The
-default roles are `admin` and `swiftoperator`, but others may be used:
+You may optionally override the default list of valid operator roles in the
+`advanced.config` file, or the old-style `app.config` file. The default roles
```

Comment:
Elsewhere we use "old-style" to describe both advanced.config and app.config, reasonably enough; here we use it differently. Strongly suggest dropping it here.